aws_marketplace/using_model_packages/data_quality_monitoring/monitoring_data_quality_of_models.ipynb

```diff
@@ -0,0 +1,776 @@
+{
```

Comment:
<p>It also makes make the notebook more interesting if you describe what are the suggested constraints.</p><p>You can add the plot of baseline distribution above and print out the suggested constraints. Then, </p><p>demonstrate why the suggested constraints make sense</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2024/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>